pkgs/by-name/mi/mitra/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitea,
+  production ? true,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mitra";
+  version = "3.9.0";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "silverpill";
+    repo = "mitra";
+    rev = "v${version}";
+    hash = "sha256-reBG9h3jI4ONxYIwM2QdXlTC8ohmSrPm18sLOeI/2wY=";
+  };
+
+  cargoLock = {
+    lockFile = "${src.outPath}/Cargo.lock";
+    outputHashes = {
+      "monero-rpc-0.4.0" = "sha256-1+BcVdOLwZ2m/xNMv82WQWlQ1uvrvusVyBiM/5hx20Q=";
+      "postgres_query-0.3.3" = "sha256-62yQpnOHUY/a6dU+6HXotxMgvlOLRJ6D0CN1HXCXfg4=";
+      "serde_qs-0.13.0" = "sha256-Q41qtmgdggi+Txyd6neL0IwT2owF10OiPyhBXfwQbug=";
+    };
+  };
+
+  # MEMO: mitra v3.9.0 tests failed with cargo option, "--offline"
+  doCheck = false;
+
+  RUSTFLAGS = [
+    # MEMO: mitra use ammonia crate with unstable rustc flag
+    "--cfg=ammonia_unstable"
+  ];
+
+  buildFeatures =
```

Comment:
Is there a reason to not have production enabled?

Reply:
Probably no. Thank you for checking...!!
resolved: 8f6ebec4dd22acbdce6e70a2d15b74bacc31ee56